### PR TITLE
Editing docstring for Label Model

### PR DESCRIPTION
**Pull request by ajratner**
## Description of proposed changes
Adds additional clarifying detail to the `LabelModel` docstring.

## Related issue(s)
Fixes # 1462

## Test plan
N/A

## Checklist
* [x] I have read the **CONTRIBUTING** document.
* [x] I have updated the documentation accordingly.
* [x] I have added tests to cover my changes.
* [x] I have run `tox -e complex` and/or `tox -e spark` if appropriate.
* [x] All new and existing tests passed.

### CODE REVIEW

File: snorkel/labeling/model/label_model.py [hunk 1]
```diff
@@ -87,7 +87,22 @@ class _CliqueData(NamedTuple):
 
 
 class LabelModel(nn.Module):
-    """A conditionally independent LabelModel to learn LF weights and assign training labels.
+    r"""A model for learning the LF accuracies and combining their output labels.
+
+    This class estimates a model of the labeling functions' conditional probabilities
```
Comment (brahmaneya) [inline snorkel/labeling/model/label_model.py]: estimates a model -> learns a model?
correctly outputting the true -> outputting the true

File: snorkel/labeling/model/label_model.py [hunk 2]
```diff
@@ -87,7 +87,22 @@ class _CliqueData(NamedTuple):
...
+    of correctly outputting the true (unobserved) label `Y`, `P(\lf | Y)`, and uses
+    this learned model to re-weight and combine their output labels.
+
+    This class is based on the approach in [Training Complex Models with Multi-Task
+    Weak Supervision](https://arxiv.org/abs/1810.02840), published in AAAI'19. In this
+    approach, we compute the inverse generalized covariance matrix of the junction tree
+    of a given LF dependency graph, and perform a matrix completion-style approach with
+    respect to these empirical statistics. The result is an estimate of the conditional
+    LF probabilities, `P(\lf | Y)`, which are then the parameters of the label model
```
Comment (brahmaneya) [inline snorkel/labeling/model/label_model.py]: then set as the parameters of...
